samples/place-autocomplete-basic-map/index.html

```diff
@@ -11,22 +11,31 @@
 
     <link rel="stylesheet" type="text/css" href="./style.css" />
     <script type="module" src="./index.js"></script>
+    <!-- prettier-ignore -->
+    <script>
+        (g=>{var h,a,k,p="The Google Maps JavaScript API",c="google",l="importLibrary",q="__ib__",m=document,b=window;b=b[c]||(b[c]={});var d=b.maps||(b.maps={}),r=new Set,e=new URLSearchParams,u=()=>h||(h=new Promise(async(f,n)=>{await (a=m.createElement("script"));e.set("libraries",[...r]+"");for(k in g)e.set(k.replace(/[A-Z]/g,t=>"_"+t[0].toLowerCase()),g[k]);e.set("callback",c+".maps."+q);a.src=`https://maps.${c}apis.com/maps/api/js?`+e;d[q]=f;a.onerror=()=>h=n(Error(p+" could not load."));a.nonce=m.querySelector("script[nonce]")?.nonce||"";m.head.append(a)}));d[l]?console.warn(p+" only loads once. Ignoring:",g):d[l]=(f,...n)=>r.add(f)&&u().then(()=>d[l](f,...n))})
+        ({key: "AIzaSyA6myHzS10YXdcazAFalmXvDkrYCp5cLc8", v: "weekly"});
+    </script>
   </head>
   <body>
     <!-- [START maps_place_autocomplete_basic_map_add] -->
-    <div id="map-container"></div>
-    <gmp-basic-place-autocomplete
-      slot="control-inline-start-block-start"
-    ></gmp-basic-place-autocomplete>
+    <gmp-map zoom="12" center="37.4220656,-122.0840897" map-id="DEMO_MAP_ID">
+      <gmp-basic-place-autocomplete
+        slot="control-inline-start-block-start"></gmp-basic-place-autocomplete>
+    </gmp-map>
     <!-- [END maps_place_autocomplete_basic_map_add] -->
-    <gmp-place-details-compact orientation="horizontal">
+    <gmp-place-details-compact
+      orientation="horizontal"
+      style="width: 400px;
```

Comment:
might be good to comment why inline styles are being used here (esp noting because this element will be moving inside the DOM once JS loads).